ftplugin/purescript_pscide.vim

```diff
@@ -57,6 +53,10 @@ if !exists("g:psc_ide_omnicompletion_prefix_filter")
   let g:psc_ide_omnicompletion_prefix_filter = v:true
 endif
 
+if !exists("g:psc_ide_server_runner")
+    let g:psc_ide_server_runner = []
```

Comment:
It looks like you're using this property as a prefix to `purs` command. Rather than that, I wonder if it's better to replace that with the `purs` command string with a function which returns it. Default to "purs", but enable explicitly setting it to something, like "yarn --something" or "/absolute/path/to/purs".